bigtable/integration_test.go

```diff
@@ -281,6 +281,62 @@ func TestIntegration_ReadRowList(t *testing.T) {
 	}
 }
 
+func TestIntegration_UpdateFamilyValueType(t *testing.T) {
+	testEnv, err := NewIntegrationEnv()
+	if err != nil {
+		t.Fatalf("IntegrationEnv: %v", err)
+	}
+	defer testEnv.Close()
+
+	if !testEnv.Config().UseProd {
+		t.Skip("emulator doesn't support update column family operation")
+	}
+
+	timeout := 15 * time.Minute
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+
+	adminClient, err := testEnv.NewAdminClient()
+	if err != nil {
+		t.Fatalf("NewAdminClient: %v", err)
+	}
+	defer adminClient.Close()
```

Comment:
Please use t.Cleanup instead of defer.
Currently, tests do not run in parallel but if in future they are made to run in parallel, defer won't work properly.
More info:
 https://brandur.org/fragments/go-prefer-t-cleanup-with-parallel-subtests#:~:text=Like%20defer%20%2C%20t.,Parallel%20%2C%20it%20behaves%20as%20expected.

https://stackoverflow.com/questions/61609085/what-is-useful-for-t-cleanup